upgrade/helpers/tasks.py

```diff
@@ -337,6 +339,17 @@ def post_upgrade_test_tasks(sat_host, cap_host=None):
     # Execute capsule sync task , after the upgrade is completed
     if cap_host:
         execute(capsule_sync, cap_host, host=sat_host)
+    # Execute task for template changes required for discovery feature
```

Comment:
Give one more tab here, indentation looks bad

Reply:
That shouldn't be in the if condition. will make it right